codecov.yml

```diff
@@ -23,4 +22,4 @@ coverage:
     project: false
 flag_management:
   default_rules:
-    carryforward: true
+    carryforward: false
```

Comment:
I initially had this as a fallback when codecov upload fails, but It was not the best of ideas, and now we don't need it anymore.

Reply:
I'm not entirely certain what this flag actually does - should I care? Is there documentation I can peek at to learn if I do need to care?

Reply:
Yup https://docs.codecov.com/docs/carryforward-flags . Turns out we don't need it, and I've been using it wrong.